hysds/job_worker.py

```diff
@@ -1438,6 +1438,26 @@ def handler(signum, frame):
             "celery_hostname": run_job.request.hostname,
         }
 
+    # overwrite message if _alt_msg.txt was dumped
+    msg = list()
+    alt_msg_file = os.path.join(job_dir, "_alt_msg.txt")
+    if os.path.exists(alt_msg_file):
+        with open(alt_msg_file) as f:
+            msgs = f.readlines()
+            logger.info("Got alternate info message: %s" % msgs)
+            for m in msgs:
+                msg.append(get_short_error(m))
+
+        job_status_json["msg"] = msg
+
+    # overwrite msg_details if _alt_msg_details.txt was dumped
+    alt_msg_details_file = os.path.join(job_dir, "_alt_msg_details.txt")
```

Comment:
```suggestion
    msg_details = None
    alt_msg_details_file = os.path.join(job_dir, "_alt_msg_details.txt")
```

Reply:
We should probably initialize `msg_details` so that line 1512 doesn't croak with an undefined variable error?

Reply:
Thanks for catching that @pymonger. I've updated the code per your suggestion.